CHANGELOG.md

```diff
@@ -1,5 +1,126 @@
 # Changelog
 
+## unreleased
+
+[Full Changelog](https://github.com/delta-io/delta-kernel-rs/compare/v0.4.0...HEAD)
+
+**API Changes**
+
+*Breaking*
+
+1. `Expression::Column(String)` is now `Expression::Column(ColumnName)` [\#400]
+2. delta_kernel_ffi::expressions moved into two modules:
+   `delta_kernel_ffi::expressions::engine` and `delta_kernel_ffi::expressions::kernel` [\#363]
+3. FFI: removed (hazardous) `impl From` for `KernelStringSlize` and added `unsafe` constructor
+   instead [\#441]
+4. Moved `LogSegment` into its own module (`log_segment::LogSegment`) [\#438]
+5. `DvInfo` now implements `Clone`, `PartialEq`, and `Eq` [\#468]
+6. `Stats` now implements `Debug`, `Clone`, `PartialEq`, and `Eq` [\#468]
```

Comment:
These are not breaking changes? At least, our PR template states:
> Note that _new_ public APIs are not considered breaking.

Reply:
ah good catch thanks!